### PR TITLE
Update marketing_operations.md

### PR DESCRIPTION
**Pull request by ebrodymoore**
Adding how we deal with requests to delete information from Sourcegraph systems

### CODE REVIEW

File: handbook/marketing/marketing_operations.md [hunk 1]
```diff
@@ -35,6 +35,6 @@ All major lead generation-related events (demo requests, webinars attended, crea
 
 To delete a contact requesting to be deleted from our systems:
 
-- Review our list of [data sources](https://about.sourcegraph.com/handbook/bizops#data-sources) and delete the contact from each source
-- Take and log [screenshots](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) of query in the [Data Processing Officer folder](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V)
+- Review our list of [data sources](https://about.sourcegraph.com/handbook/bizops#data-sources) and delete the contact from each source. 
+- Take and log [screenshots](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) in the [Data Processing Officer folder](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) of attempt to find contact in the system.
```
Comment (aileenrose) [inline handbook/marketing/marketing_operations.md]:
```suggestion
- Take and log [screenshots](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) in the [Data Protection Officer folder](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) of attempt to find contact in the system. 
```

File: handbook/marketing/marketing_operations.md [hunk 2]
```diff
@@ -35,6 +35,6 @@ All major lead generation-related events (demo requests, webinars attended, crea
 
 To delete a contact requesting to be deleted from our systems:
 
-- Review our list of [data sources](https://about.sourcegraph.com/handbook/bizops#data-sources) and delete the contact from each source
-- Take and log [screenshots](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) of query in the [Data Processing Officer folder](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V)
+- Review our list of [data sources](https://about.sourcegraph.com/handbook/bizops#data-sources) and delete the contact from each source. 
+- Take and log [screenshots](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) in the [Data Processing Officer folder](https://drive.google.com/drive/folders/1974oAceBuVXt-3a1ULI2LgBHk6gID-3V) of attempt to find contact in the system. 
 - Email contact from personal Sourcegraph email with the following `We have received your CCPA request to be deleted from Sourcegraph systems. Your information has been permanently wiped from all our systems (screenshot attached), and as a result you won't receive any additional emails from us. [PERSONALIZED SENTENCE based on situation]`.
```
Comment (aileenrose) [inline handbook/marketing/marketing_operations.md]:
```suggestion
- Email contact from personal Sourcegraph email with the following `We have received your [CCPA|GDPR] request to be deleted from Sourcegraph systems. Your information has been permanently wiped from all our systems (screenshot attached), and as a result you won't receive any additional emails from us. [PERSONALIZED SENTENCE based on situation]`.  
```

Comment (aileenrose) [reply handbook/marketing/marketing_operations.md]: communications include phone calls.

**Comment by dadlerj** [comment]
LGTM, thanks!

General question @aileenrose what is best practice for ensuring future outbound marketing avoids these individuals? Just manually reviewing every screenshot in that folder first?

**Comment by aileenrose** [comment]
> LGTM, thanks!
> 
> General question @aileenrose what is best practice for ensuring future outbound marketing avoids these individuals? Just manually reviewing every screenshot in that folder first?

Now that we have an SDR!, let's make sure they are identified properly.  In the future, outbound leads would be nurtured in a separate database i.e. outbound.io or similar

**Comment by rootofnetworking-web** [comment]
I love it.

**Pull request merged**